independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -115,6 +116,10 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                             invocation.getQuarkusProject().getCodestartResourceLoaders());
             final CodestartProjectDefinition projectDefinition = catalog.createProject(input);
             projectDefinition.generate(invocation.getQuarkusProject().getProjectDirPath());
+
+            // TODO: Add submodule to maven parent - Include https://github.com/fabric8-launcher/maven-model-helper for this purpose
+            // TODO: Also do we want to rollback if failing to add to the parent ?
```

Comment:
That's a good point. I think a warning could also be enough to let the user add it manually?